docker/.env.example

```
@@ -0,0 +1,22 @@
+MY_UID=1000
+MY_GID=1000
+
+# Profile Service Configuration
+# Path where the profile service stores its database
+DATABASE_PATH=/data/profiles.db
+
+# reverse proxy domain
+DOMAIN=your-domain.com
+SERVER_IP=your.server.ip.address
+
+CIRCLES_PLUGIN_DISABLED=false # Set to true to disable the Circles Nethermind plugin when running a reindex job
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)